data_api/__init__.py

```diff
@@ -0,0 +1,2 @@
+NAME = 'rapidpro-dw'
```

Comment:
thoughts on extending to `'rapidpro-data-warehouse'`? i'm generally acronym averse...

Reply:
sure thing